src/metabase/driver/sql_jdbc/csv.clj

```diff
@@ -0,0 +1,42 @@
+(ns metabase.driver.sql-jdbc.csv
+  (:require
+   [honey.sql :as sql]
+   [metabase.query-processor.writeback :as qp.writeback]))
+
+(defn file-schema
+  "Returns a list of data for all the columns."
+  [file]
+  ;; WIP: dummy data for now
+  [{:csv-name      "column_name"
```

Comment:
I don't need `csv-name` to create the table as long as `database-name` is in there. But it might be helpful for displaying error messages during parsing.